graviti/manager/dataset.py

```diff
@@ -11,20 +11,27 @@
 
 from graviti.dataframe import DataFrame
 from graviti.exception import ResourceNotExistError
-from graviti.manager.branch import BranchManager
+from graviti.manager.branch import Branch, BranchManager
 from graviti.manager.commit import Commit, CommitManager
 from graviti.manager.draft import DraftManager
 from graviti.manager.lazy import LazyPagingList
-from graviti.manager.tag import TagManager
+from graviti.manager.tag import Tag, TagManager
 from graviti.openapi import (
     create_dataset,
     delete_dataset,
     get_dataset,
+    get_revision,
     list_datasets,
     update_dataset,
 )
 from graviti.utility import ReprMixin, ReprType, UserMutableMapping
 
+_REVISION_TYPES = {
```

Comment:
will it be better to use python `enum.Enum`?

Reply:
It has been modified. Should we use `enum` instead of `dict` in similar scenarios? What are the benefits?